client/src/components/BookingCards/BookingCard.tsx

```diff
@@ -0,0 +1,116 @@
+/* eslint-disable prettier/prettier */
+import React  from "react";
+import useStyles from "./useStyles";
+import { 
+  Card, 
+  CardHeader, 
+  CardContent, 
+  IconButton, 
+  Typography, 
+  Link,
+  Box, 
+  Grid
+} from "@material-ui/core";
+import { Settings } from "@material-ui/icons";
+
+const mockImg = 'https://badairies.co.uk/assets/admin/plugins/images/users/4.jpg';
```

Comment:
We should have profile picture support in the application, why do we need this?

Reply:
Will we keep propfile photo property in User model? Because in the getRequests controller I am fetching data from Profile model. I think that profilePhoto should be in Profile model for better consistency.

Reply:
I was waiting to define that issue.